README.md

```diff
@@ -43,9 +43,9 @@ Basic usage for executing a lane in the root directory without arguments.
 
 ```
 - uses: actions/checkout@v2
-- uses: actions/setup-ruby@v1
+- uses: ruby/setup-ruby@v1
   with:
-    ruby-version: '2.7.2'
+    ruby-version: 3.0
```

Comment:
Instead of bumping ruby to 3.0, it's probably better to try the latest v2 version 2.7.6